python/tank/platform/engine.py

```diff
@@ -267,8 +267,6 @@ def __init__(self, tk, context, engine_instance_name, env):
             self.log_debug("Metrics dispatcher started.")
 
         self.log_debug("Init complete: %s" % self)
-        # Emit an event for the software being launched
-        self.log_metric("Launched Software")
```

Comment:
@robblau Note that moving this to the `tk-multi-launchapp` we lose the ability to log Host App and Host App Version from this event ( since engine are not yet started in `tk-multi-launchapp` )